packages/react/src/components/Code/index.tsx

```diff
@@ -26,7 +26,7 @@ type Props = {
   scrollable?: boolean;
   label?: ContentNode;
   allowCopy?: boolean;
-  copyButtonProps?: ButtonProps;
+  copyButtonProps?: CopyButtonProps;
```

Comment:
We generally don't rely on the export types:

```suggestion
  copyButtonProps?: React.ComponentProps<typeof CopyButtonProps>;
```